modules/openy_gc_auth/modules/openy_gc_auth_yusa/openy_gc_auth_yusa.module

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Main file for openy_gc_auth_yusa module.
+ */
+
+/**
+ * Implements hook_mail().
+ */
+function openy_gc_auth_yusa_mail($key, &$message, $params) {
+  switch ($key) {
+    case 'email_verification':
```

Comment:
This code does the same as this: https://github.com/fivejars/openy_gated_content/blob/master/modules/openy_gc_auth/modules/openy_gc_auth_custom/openy_gc_auth_custom.module#L11
We have to add different email key or delete it.